website/content/docs/release-notes/consul-ecs/v0_5_x.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+page_title: 0.5.x
+description: >-
+  Consul ECS release notes for version 0.5.x
+---
+
+# Consul ECS 0.5.x
+
+## Release Highlights
+
+- **Audit Logging (Enterprise) :** This feature supports audit logging by providing an `audit-logging` flag, providing a log of authenticated events if enabled.
+
+- **Support Consul AWS IAM auth method :** This feature provides support obtaining ACL tokens from Consul's AWS IAM auth method.
```

Comment:
```suggestion
- **AWS IAM Auth Method :** This feature provides support for Consul's AWS IAM auth method. This allows AWS IAM roles and users to authenticate with Consul to obtain ACL tokens.
```

Reply:
Added new phrasing to match @cthain suggestion and linked to appropriate section for configuration info in [latest commit](https://github.com/hashicorp/consul/pull/14010/commits/d5f334cf1ee381d5d5b60a4bc33d8f5f99ce9f17)